blog/admin.py

```diff
@@ -0,0 +1,7 @@
+"""The admin configuration for the Blog application."""
+
+
+from django.contrib import admin
+from .models import Post
+
+admin.site.register(Post)
```

Comment:
Why don't you make this a part of #33?

Reply:
Yes, I will combine these pull requests.

Reply:
> Why don't you make this a part of #33?

I should then close this pull request.